x-pack/plugins/security_solution/public/detections/components/rules/step_define_rule/index.tsx

```diff
@@ -489,6 +499,46 @@ const StepDefineRuleComponent: FC<StepDefineRuleProps> = ({
     ]
   );
 
+  const GroupByChildren = useCallback(
+    ({ groupByRadioSelection, groupByDurationUnit, groupByDurationValue }) => (
+      <EuiRadioGroup
+        disabled={
+          !license.isAtLeast(minimumLicenseForSuppression) ||
+          groupByFields == null ||
+          groupByFields.length === 0
+        }
+        idSelected={groupByRadioSelection.value}
+        options={[
+          {
+            id: GroupByOptions.PerRuleExecution,
+            label: 'Per rule execution',
+          },
+          {
+            id: GroupByOptions.PerTimePeriod,
+            label: (
+              <>
+                {`Per time period`}
+                <DurationInput
+                  durationValueField={groupByDurationValue}
+                  durationUnitField={groupByDurationUnit}
+                  isDisabled={
+                    groupByFields?.length === 0 ||
+                    groupByRadioSelection.value !== GroupByOptions.PerTimePeriod
```

Comment:
Whoops -- one even more corner case here with regards to being on basic license and then duplicating a prebuilt rule that has `alert_suppression`:


<p align="center">
  <img width="500" src="https://user-images.githubusercontent.com/2946766/214692097-3ad034f5-b56f-4a25-a055-9765ed0df12c.png" />
</p>



```suggestion
                  isDisabled={                  
                    !license.isAtLeast(minimumLicenseForSuppression) ||
                    groupByFields?.length === 0 ||
                    groupByRadioSelection.value !== GroupByOptions.PerTimePeriod
```